assets/javascript/game.js

```diff
@@ -4,42 +4,61 @@ var guessWrong = 0;
 var maxGuesses = 7;
 var winCount = 0;
 var lossCount = 0;
-var wordBank = ["Algebra", "Poodle", "Citizen", "Pizzaz", "Cooperate", "Munchkin", "Flower", "Supercalifragilisticexpialidocious"];
+var wordBank = ['Algebra', 'Poodle', 'Citizen', 'Pizzaz', 'Cooperate', 'Munchkin', 'Flower', 'Supercalifragilisticexpialidocious'];
 
-//Pull from wordBank to give puzzle word
+// Pull from wordBank to give puzzle word, and make sure all buttons are enabled
```

Comment:
add some comment you will not i added a space after the `//` this is generally a best practice